README.md

```diff
@@ -23,6 +23,7 @@ t.set_colors({
 
 t.build("mytheme.min.js")
 ```
+[![Run on Repl.it](https://user-images.githubusercontent.com/27065646/92304596-bf719b00-ef7f-11ea-987f-2c1f3c323088.png)](https://repl.it/github/sherlock-project/sherlock)
```

Comment:
Please change this link to the Reflux repository: [https://github.com/IreTheKID/Reflux](https://github.com/IreTheKID/Reflux)